README.md

```diff
@@ -77,10 +77,11 @@ static_metadata:
 
 The matrix below lists the versions of Prometheus Server and other dependencies that have been qualified to work with releases of `stackdriver-prometheus-sidecar`.
 
-| sidecar version | **Prometheus 2.4.3** | **Prometheus 2.5.x** | **Prometheus 2.6.x** |
-|------------|------------------|-------------------|-------------------|
-| **0.2.x**  |        ✓         |         -         |         -         |
-| **0.3.x**  |        ✓         |         -         |         ✓         |
+| sidecar version | **Prometheus 2.4.3**| **Prometheus 2.5.x**| **Prometheus 2.6.x**|
+|-----------------|---------------------|---------------------|---------------------|
+| **0.2.x**       |          ✓          |          -          |          -          |
+| **0.3.x**       |          ✓          |          -          |          ✓          |
+| **0.4.x**       |          -          |          -          |          ✓          |
```

Comment:
Does 2.4.3 not work, or you haven't tested it?

Reply:
I didn't test against 2.4.3 yet, so I leave it empty there.

Reply:
"-" means "no" right now. Can you make it something like ? and then add below "?: not verified yet"? Thanks.

Reply:
Updated. Please take a look and let me know what do you think, thanks.